internal/policy/policy.go

```diff
@@ -611,21 +609,21 @@ func (s *State) Commit(ctx context.Context, repo *git.Repository, commitMessage
 		return err
 	}
 
-	ref, err := repo.Reference(plumbing.ReferenceName(PolicyRef), true)
+	ref, err := repo.Reference(plumbing.ReferenceName(PolicyStagingRef), true)
```

Comment:
I thought that since all state changes should be routed through the policy staging namespace, there would be no need to commit changes to the policy namespace.

Reply:
I think it depends on the workflow we collectively decide on, so you're right. I'd like to hear from other maintainers here.

Reply:
I agree that the current version of policy staging is quite painful to use, especially with multiple contributors creating changes to the policy state. 

I was thinking about addressing these issues, and https://github.com/gittuf/gittuf/pull/260#issuecomment-1962047511,  in a follow-up PR since if we can merge this simple, bare-bones version of policy-staging, making it just work(albeit in a clunky manner), other problems can start to be fixed, like those listed here https://github.com/gittuf/gittuf/issues/230#issuecomment-1961448955.

Reply:
I think treating staging refs as any other branch is fine. The only thing I want us to think through is what happens when merging multiple policy changes. If we have a good story there, we should be fine.